runtime/op/uniq/uniq.go

```diff
@@ -26,7 +26,7 @@ func New(octx *op.Context, parent zbuf.Puller, cflag bool) *Op {
 	}
 }
 
-func (o *Op) wrap(t *zed.Value) *zed.Value {
+func (o *Op) wrap(t *zed.Value) zed.Value {
```

Comment:
Why not nix the rest of the pointers in this file including `Op.last`, the `t` arg here and in appendUniq?

Reply:
I tried but there's a comparison between `Op.last` and `nil` in `Op.Pull` that appears to be significant.